server/src/catalog.rs

```diff
@@ -138,7 +142,7 @@ fn is_dir_entry_a_table(dir_entry: &DirEntry) -> bool {
 }
 
 fn is_dir_entry_a_parquet_file(dir_entry: &DirEntry) -> bool {
-    let mut file = File::open(dir_entry.path()).unwrap();
+    let mut file = OpenOptions::new().write(true).open(dir_entry.path()).unwrap();
```

Comment:
Is this change relevant to tracing? If not it should be reverted and handled in a separate branch. As far as i remember this is the fix to the Windows issue so it should probably be in an individual bug fix PR.

Reply:
Fixed

Reply:
@skejserjensen Should we fix this bug now or will it be handled later as part of another change?

Reply:
I think we should fix it in a separate branch now as I do not know when the `catalog` module will be updated. Also we should probably include a comment that documents why a file that is only read is opened with `write(true)`.

Reply:
Alright, I can do this once the tracing branch is merged.